packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -318,4 +318,28 @@ class SystemChannels {
     'flutter/restoration',
     StandardMethodCodec(),
   );
+ 
+  /// A [MethodChannel] for installing and managing dynamic features.
+  ///
+  /// The following outgoing methods are defined for this channel (invoked using
+  /// [OptionalMethodChannel.invokeMethod]):
+  ///
+  ///  * `installDynamicFeature`: Requests that a dynamic feature identified by
```

Comment:
You wanna cross-reference the dart API that does the same thing on deferred libraries?